docs/DafnyRef/Options.txt

```diff
@@ -360,30 +360,30 @@ Usage: Dafny [ option ... ] [ filename ... ]
   
   ---- Test generation options -----------------------------------------------
   
+  /warnDeadCode
```

Comment:
FYI we've got a new `dafny` CLI that supports top-level verbs now, and the preferred way to invoke this feature is now `dafny generate-tests <options...>`. You'll have to refactor a bit when you merge from the latest, but the nice things is that you can avoid all the `/generate` prefixes. :)

Reply:
Yes, this feature is amazing! I already updated that part for the PR but will sync again in the next few days